.travis.yml

```diff
@@ -46,6 +46,8 @@ before_install:
 
 install:
   - make arm_sdk_install
+  - export TOOLCHAINPATH=$PWD/tools/gcc-arm-none-eabi-7-2017-q4-major/bin/
```

Comment:
Having the separate `TOOLCHAIN` variable, no.

The Ruby scripts do expect to find arm-gcc on the path, this may change if my method mentioned in https://github.com/betaflight/betaflight/pull/4970#discussion_r162162502 works.